src/naf-dialog-adapter.js

```diff
@@ -99,6 +102,62 @@ export default class DialogAdapter {
     this._onOccupantMessage = messageListener;
   }
 
+  async getServerStats() {
+    const result = {};
+    try {
+      if (this._sendTransport && !this._sendTransport._closed && this._sendTransport._connectionState == "connected") {
```

Comment:
Since these if statements are relatively complicated with multiple checks, I'd like to see a short comment of what each one is doing.

Reply:
Not to be pedantic, but I'm curious about what type of comment you would expect here. The code seems pretty self-documenting to me. I just read this as "if we have a _sendTransport, and the transport is not closed, and the transport's connection state is connected". 
Perhaps instead of adding a comment, you'd be satisfied with something like this, where the variable name serves as a summary of the condition?
```js
const sendTransportAvailable = (
  this._sendTransport && 
  !this._sendTransport._closed && 
  this._sendTransport._connectionState == "connected"
);
if (sendTransportAvailable) {
...
```
Generally I think we should make the code as clear as possible (which I think is already the case here, IMO) and reserve comments for "why" explanations where necessary, as opposed to "what" explanations. It would feel redundant to me to just have a comment say what the code is doing when you can make the code readable in the first place, but perhaps it's a matter of preference.

Reply:
Actually that can really be simplified. I'll update.